docs/insomnia/get-started-with-documents.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: article-detail
+title: Get Started with Documents
+category: "API Design"
+category-url: api-design
+---
+
+Design Documents (also called Documents) hold specifications, API requests, and tests. They also offer the capability to sync with Git.
+
+If you're looking for information about Collections instead (which use [Version Control Sync](/insomnia/version-control-sync)), see [Request Collections](/insomnia/request-collections).
+
+For this guide, [create an empty Document](/insomnia/design-documents) from your Dashboard. We've named our Document "petstore" as our example uses the [Pet Store API](https://petstore.swagger.io/).
+
+## Tab Overview
+
+We're going to start in the Design tab. We'll walk through each tab in order from left to right, starting with the Design tab.
+
+![Documents have three tabs: Design, Debug, and Test.](/assets/images/three-document-tabs.png)
+
+_Documents have functionality separated into three tabs: Design, Debug, and Test._
+
+Tab functionality is as follows:
+
+* [**Design**](#design-tab): Import and edit specifications, see API overview content, and preview a rendering of the API.
+* [**Debug**](#debug-tab): Create, edit, and send individual requests. Organize your requests into folders. View request responses.
+* [**Test**](#test-tab): Create test suites and individual tests for requests. Run tests individually or as a group. View test results.
+
+## Design Tab
+
+The Design tab is the first of three top-level tabs in Documents. Import your specification on this tab.
+
+### Add Your Specification
+
+We're going to import a sample file that's built into Insomnia. Click on **start from an example** (above the **Import** button), which will populate the sample Pet Store API.
+
+The Pet Store specification will populate in the center panel of the Design tab. In this panel, you can directly edit the specification.
+
+{:.alert .alert-secondary}
+**Alternative Option**: Rather than use the sample Pet Store API file, you can import from a file or URL. In the center panel of the Design tab, you'll see an **Import** button. This dropdown allows you to select between a File or URL. For a file, select the file to import from your computer. For a URL, provide the URL where the specification file lives. Otherwise, if you're starting from scratch, you can start typing right into the center panel.
```

Comment:
nit: just struck me as strange to say "Alternative Option" (which seems sorta redundant to me) rather than just "Alternative"

Reply:
I can see what you're saying, though just saying "Alternative" could be less explicit. Maybe "Alternative Import Options"? This way it becomes more explicit.